README.md

```diff
@@ -54,9 +56,42 @@ To increase logging level to include info and debug messages, set the environmen
 `DMCI_LOGLEVEL` to the desired level. Valid levels are `CRITICAL`, `ERROR`, `WARNING`, `INFO`, and
 `DEBUG`.
 
+## Installation
+```
+git clone https://github.com/metno/discovery-metadata-catalog-ingestor
+
+cd discovery-metadata-catalog-ingestor
+
+mkdir workdir
+
+cd workdir
+
+mkdir rejected
+
+```
+Initialize the config.yaml or use environment-variables.
+
+```
+dmci:
+  distributors:
+    - file
+  distributor_cache: workdir
+  rejected_jobs_path: workdir/rejected
+  max_permitted_size: 100000
+  mmd_xsl_path: path/to/mmd/xslt/mmd-to-geonorge.xsl
+  mmd_xsd_path: path/to/mmd/xsd/mmd_strict.xsd
+  path_to_parent_list: parent-uuid-list.xml
+
+pycsw:
+  csw_service_url: http://localhost
+
+file:
+  file_archive_path: workdir
+
+```
```

Comment:
also maybe `parent-uuid-list.xml` and `workdir` can be added to `.gitignore` just after 
```
# Input
/config.yaml
```
so that one can test in a working copy of the repo without having to stash changes

Reply:
It is probably practical to have the empty xml file there. I suggest to keep it in version control. `workdir` can probably be gitignore'd.